packages/list-plugins/lib/tracking/atTracking.js

```diff
@@ -0,0 +1,69 @@
+const { DateTime } = require('@keystone-alpha/fields');
+
+const composeResolveInput = (originalHook, newHook) => async params => {
+  const resolvedData = await originalHook(params);
+  return newHook({ ...params, resolvedData });
+};
+
+const _atTracking = ({ created = true, updated = true }) => ({
+  updatedAtField = 'updatedAt',
+  createdAtField = 'createdAt',
+  ...atFieldOptions
+} = {}) => ({ fields = {}, hooks = {}, ...rest }) => {
+  const dateTimeOptions = {
+    type: DateTime,
+    format: 'MM/DD/YYYY h:mm A',
+    access: {
+      read: true,
+      create: true, // TODO: revert to false when read only fields are available
+      update: true, // TODO: revert to false when read only fields are available
+    },
+    ...atFieldOptions,
+  };
+
+  if (updated) {
+    fields[updatedAtField] = {
+      ...dateTimeOptions,
+    };
+  }
+
+  if (created) {
+    fields[createdAtField] = {
+      ...dateTimeOptions,
+    };
+  }
+
+  const originalResolveInput = hooks.resolveInput || (({ resolvedData }) => resolvedData); // possible to have no hooks provided
```

Comment:
```suggestion
  const originalResolveInput = hooks.resolveInput
```
Can we deal with the no hooks situation in `composeHooks` rather than creating a noop function? 

Let's also move `originalResolveInput` under `newResolveInput` as it get's lost up here.


I also think we need to make a new issue so that hooks can accept an array or we expose a generic `composeHooks` function. It should not be the responsibility of plugin authors to do this.